Lib/inspect.py

```diff
@@ -2518,14 +2518,17 @@ def replace(self, *, name=_void, kind=_void,
     def __str__(self):
         kind = self.kind
         formatted = self._name
-
+        
```

Comment:
Trim the whitespace.

Reply:
@1st1 
Thanks, When I noticed it, you already read it. :)